### PR TITLE
feat(oui-navbar): add onclick and manual control on toggler

### PR DESCRIPTION
**Pull request by AxelPeter**
- Add `on-click` attribute
- Add `active` flag
- Add `loading` flag
- Add unit tests

### CODE REVIEW

File: packages/oui-navbar/src/index.spec.js [hunk 1]
```diff
@@ -581,6 +581,44 @@ describe("ouiNavbar", () => {
...
+                    <oui-navbar-toggler loading="$ctrl.loading"></oui-navbar-toggler>
+                </oui-navbar>`);
+                const scope = component.scope();
+                const toggler = angular.element(component[0].querySelector(".oui-navbar-toggler_button"));
+
+                expect(toggler.hasClass("ng-hide")).toBeFalsy(); // Check when undefined
+
+                scope.$ctrl.loading = true;
+                scope.$apply();
+                expect(toggler.hasClass("ng-hide")).toBeTruthy();
```
Comment (euhmeuh) [inline packages/oui-navbar/src/index.spec.js]: Maybe you should check for the visibility of the spinner too?

File: packages/oui-navbar/README.md [hunk 2]
```diff
@@ -298,12 +298,41 @@ It defines the menu in reponsive mode. It will be visible only for screen resolu
...
 <oui-navbar active-link="lorem">
     <oui-navbar-toggler links="$ctrl.togglerLinks"></oui-navbar-toggler>
 </oui-navbar>
 ```
 
+#### As toggler button
+
+```html:preview
+<div class="oui-doc-preview-only">
+    <p>
```
[the remark this answers is not in the document]
Comment (AxelPeter) [reply packages/oui-navbar/README.md]: For margin. 
This part is `oui-doc-preview-only`

File: packages/oui-navbar/src/index.spec.js (hunk 1 above)
Comment (AxelPeter) [reply packages/oui-navbar/src/index.spec.js]: I'll check that

File: packages/oui-navbar/README.md [hunk 3]
```diff
@@ -298,12 +298,41 @@ It defines the menu in reponsive mode. It will be visible only for screen resolu
...
+        <button class="oui-button oui-button_primary" type="text" ng-class="{
```
Comment (antleblanc) [inline packages/oui-navbar/README.md]: You mean `type="button"` isn't it?

Comment (AxelPeter) [reply packages/oui-navbar/README.md]: I must be tired...

Comment (antleblanc) [reply packages/oui-navbar/README.md]: Take care of you 💋

**Pull request merged**